view/frontend/templates/js/replacejs.phtml

```diff
@@ -753,85 +755,56 @@ ob_start();
                         var BC = BoltCheckout.configure(cart, hintBarrier.promise, callbacks);
                 });
             } else {
-            // define the params sent with the request variable
-            var params = [];
-            params.push('form_key=' + $('[name="form_key"]').val());
-            params = params.join('&');
-
             var hintBarrier = boltBarrier();
+            var cartBarrier = boltBarrier();
 
-            cart = new Promise(function (resolve, reject) {
-
-                createRequest = true;
-
-                // send create order request
-                $.get(settings.create_order_url, params)
-                    .done(function (data) {
+            customerData.get('boltcart').subscribe(function(data) {
```

Comment:
just to my knowledge, where this `boltcart` thing get updated?

Reply:
This is magento section, an implementation that allows saving data in browser local storage.
Magento provides invalidation by time/user actions. When data isn't valid magento calls our backend code to get new value.
Backend realization:
https://github.com/BoltApp/bolt-magento2/pull/738
Some additional information
https://magento.stackexchange.com/questions/112948/magento-2-how-do-customer-sections-sections-xml-work